### DISCUSSION TITLE
Finished German Translation

### DISCUSSION THREAD

**Discussion by akaydev-coder** [Qalculate/libqalculate#276] [Show and tell]
Finished part 1 for german translation (libqalculate/po/de.po) - moving now over to 
@hanna-kn should I switch to libqalculate/po-def/ or should I move to qalculate-gtk as you've mentioned in your mail?

**Comment by hanna-kn** [comment]
It dependence on if you prioritize the GUI or not. The translation of po-defs is more complicated (it is however not necessary to translate all aspects).

I should mention that I plan to release the next version 2021-05-24.

**Reply by akaydev-coder** [reply]
Thanks for quick reply. I prefer the gtk version. ( using it in Windows ) 
I hope to finish it before you‘re releasing the next version.
